nixos/modules/services/backup/bacula.nix

```diff
@@ -517,23 +533,23 @@ in {
       after = [ "network.target" "postgresql.service" ];
       description = "Bacula Director Daemon";
       wantedBy = [ "multi-user.target" ];
-      path = [ pkgs.bacula ];
+      path = [ pkgs.bacula pkgs.gawk pkgs.mtx pkgs.mt-st pkgs.su ];
```

Comment:
As above, but the director daemon also needs to be able to su to postgres to configure the database for the first time.